src/org/gk/elv/EntityLevelView.java

```diff
@@ -470,36 +472,33 @@ public void deleteInstance(GKInstance instance) {
     
     /**
      * Switch the type of a GKInstance.
-     * @param cls
-     * @param instance
+     * @param instance the instance whose schemaClass has been changed
      */
-    public void switchedType(GKSchemaClass oldCls,
-                             GKInstance instance) {
-        objectListPane.switchedType(oldCls,
-                                    instance);
-        zoomableEditor.switchedType(oldCls, instance);
+    public void switchedType(GKInstance instance) {
```

Comment:
Please see my other comment regarding this method. oldCls may be needed, e.g., to generate an information. Please keep it.